tests/test_client_branding_slugify.py

```
@@ -0,0 +1,41 @@
+and replacing spaces, leaving `../` and OS path separators free to
+escape the branding directory.
+"""
+
+from __future__ import annotations
+
+import pytest
+
+
+class TestSlugifyClientName:
+
+    @pytest.mark.parametrize("raw,expected", [
```

Ruff (0.15.12)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)